.github/workflows/test-client.yml

```diff
@@ -45,8 +54,23 @@ jobs:
       - run: ${{ inputs.test-script }}
         timeout-minutes: 60
 
-      - uses: DevExpress/testcafe-build-system/actions/set-status@main
+      - uses: Aleksey28/testcafe-build-system/actions/save-matrix-status@main
```

Comment:
Is it a development artifact?

Reply:
It was temporary until https://github.com/DevExpress/testcafe-build-system/pull/22 wasn't merged.